clients/geth/specular/README.md

```diff
@@ -2,6 +2,7 @@
 
 ## Build
 
+First setup L1 and then follow following steps
```

Comment:
nit move this to under `Running a local network` since it's not necessary for building; also
```suggestion
Please follow the [L1 setup process](url to relevant README) before running the steps below.
```

Reply:
Moved under "Running a local network"